mayhem/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM --platform=linux/amd64 ubuntu:20.04 as builder
```

Comment:
Can we revisit the sed command here? Is there a better way to get this to build?